lib/core/src/getUtil.cpp

```diff
@@ -369,19 +369,7 @@ initCondForGet( rcComm_t *conn, rodsArguments_t *rodsArgs,
         conn->fileRestart.flags = FILE_RESTART_ON;
         rstrcpy(conn->fileRestart.infoFile, rodsArgs->lfrestartFileString, MAX_NAME_LEN);
     }
-    // =-=-=-=-=-=-=-
-    // JMC - backport 4604
-    if ( rodsArgs->rlock == True ) {
-        addKeyVal( &dataObjOprInp->condInput, LOCK_TYPE_KW, READ_LOCK_TYPE );
```

Comment:
Is anything using `LOCK_TYPE_KW` or `READ_LOCK_TYPE`? If not, let's remove those too.